src/openwith.js

```diff
@@ -16,21 +16,31 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 OME.setOpenWithUrlProvider("omero_iviewer", function(selected, url) {
+    selected_count = selected.length;
+    // last selected image
+    last_selected = selected[selected_count-1];
     // Add image Id to url
-    url += selected[0].id + "/";
+    url += last_selected.id + "/";
```

Comment:
I guess the questions (for now) is: which one in the list of multi-selected should is the initial one to be shown? should it be the fist or remain the last (as is now) and should the natural order be preserved or should the initial image be placed first in the list of thumbnails.